test/integration/dind_test.js

```diff
@@ -0,0 +1,40 @@
+import assert from 'assert';
+import DockerWorker from '../dockerworker';
+import TestWorker from '../testworker';
+
+suite('use dind-service', () => {
+  let worker;
+  setup(async () => {
+    worker = new TestWorker(DockerWorker);
+    await worker.launch();
+  });
+
+  teardown(async () => {
+    if (worker) {
+      await worker.terminate();
+      worker = null;
+    }
+  });
+
+  test('run docker in docker', async () => {
+    let result = await worker.postToQueue({
+      payload: {
+        image: 'taskcluster/dind-test:v1',
+        command: [''],
```

Comment:
Agree... we can test more later... It's hard to stuff things into command, but this one is so simple...